math-native/src/main/java/org/ballerinalang/stdlib/math/nativeimpl/Constant.java

```diff
@@ -34,10 +34,4 @@
     public static final String DIVIDE_BY_ZERO_ERROR_MSG = "Division by zero occurred";
 
     public static final String OVERFLOW_ERROR_MSG = "Overflow occurred";
-
-    private static final String PACKAGE_NAME = "math";
-
-    public static final String MATH_PACKAGE_VERSION =  "1.0.1";
-
-    static final Module MATH_PACKAGE_ID = new Module(BALLERINA_BUILTIN_PKG_PREFIX, PACKAGE_NAME, MATH_PACKAGE_VERSION);
```

Comment:
The `BALLERINA_BUILTIN_PKG_PREFIX` is now an unused import, right?

This gets me to think, is spotbugs and the checkStyle enabled for this repo? If not, shall we enable them?

Reply:
Enabled in 1475b4e